### PR TITLE
Renamed brand names

### PR DESCRIPTION
**Pull request by ivanmkc**
Renamed brand names.

- Ignore lint errors since those are caused by community notebooks.
- This doesn't rename any files as that would break links. That can be a subsequent PRs if required.

### CODE REVIEW

**Comment by ivanmkc** [comment]
I would like feedback on the capitalization.

File: notebooks/community/migration/UJ5 unified AutoML Vision Video Classification.ipynb [hunk 1]
```diff
@@ -29,7 +29,7 @@
         "id": "title:migration,new"
```
Comment (alecglassford) [inline notebooks/community/migration/UJ5 unified AutoML Vision Video Classification.ipynb:29]: <p>I suggest that this title becomes:</p><p> </p><p>"Vertex AI AutoML Image Object Detection"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

File: notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb [hunk 2]
```diff
@@ -99,7 +99,7 @@
       "source": [
```
Comment (alecglassford) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb:99]: <p>It's a mouthful but I think the most unambiguous terminology would be </p><p> </p><p>(Option A) "… or Vertex AI Workbench user-managed notebooks"</p><p> </p><p>—and actually I think just the following would make sense:</p><p> </p><p>(Option B) "…&nbsp;or Vertex AI Workbench notebooks"</p><p> </p><p>(the implication being that the authentication/setup is taken care of for you in user-managed notebooks AND managed notebooks; I think this is true, but you know more about managed notebooks than me)</p><p> </p><p>Could you do another find and replace to one of my two suggestions, depending on which you think makes sense?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

File: notebooks/official/model_monitoring/model_monitoring.ipynb [hunk 3]
```diff
@@ -30,7 +30,7 @@
         "id": "fsv4jGuU89rX"
```
Comment (alecglassford) [inline notebooks/official/model_monitoring/model_monitoring.ipynb:30]: <p>Several notebooks seem to have this duplicate "Vertex AI Vertex AI" in the title.</p><p> </p><p>Can you do a find-and-replace for "Vertex AI Vertex AI" --&gt; "Vertex AI" across all files?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

File: notebooks/official/automl/automl-tabular-classification.ipynb [hunk 4]
```diff
@@ -51,7 +51,7 @@
         "id": "tvgnzT1CKxrO"
```
Comment (ivanmkc) [inline notebooks/official/automl/automl-tabular-classification.ipynb:51]: <p>I'll fix this Vertex AI Vertex AI business...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

File: notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb (hunk 2 above)
Comment (ivanmkc) [reply notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb:99]: <p>I'll go with "or Vertex AI Workbench notebooks"</p>

Comment (ivanmkc) [reply notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb:99]: Done

File: notebooks/community/migration/UJ5 unified AutoML Vision Video Classification.ipynb (hunk 1 above)
Comment (ivanmkc) [reply notebooks/community/migration/UJ5 unified AutoML Vision Video Classification.ipynb:29]: Done

File: notebooks/official/model_monitoring/model_monitoring.ipynb (hunk 3 above)
Comment (ivanmkc) [reply notebooks/official/model_monitoring/model_monitoring.ipynb]: Fixed.

**Review by alecglassford — APPROVED** [review]
LGTM

**Pull request merged**